apps/builder/app/routes/rest.assets.$name.tsx

```diff
@@ -0,0 +1,38 @@
+import type { ActionArgs } from "@remix-run/node";
+import type { Asset } from "@webstudio-is/asset-uploader";
+import { completeUpload } from "@webstudio-is/asset-uploader/index.server";
+import type { ActionData } from "~/builder/shared/assets";
+import { sentryException } from "~/shared/sentry";
+import { createAssetClient } from "~/shared/asset-client";
+
+export const action = async (
+  props: ActionArgs
+): Promise<ActionData | Array<Asset> | undefined> => {
+  const { request, params } = props;
+
+  if (params.name === undefined) {
+    throw new Error("Name is undefined");
+  }
+
+  try {
+    if (request.method === "POST") {
+      const asset = await completeUpload(
+        {
+          name: params.name,
```

Comment:
What if everything in URL is considered as not secure, so bad actor can intercept and upload other photo with knowledge of `name`
Not an issue at all anyway as I have no idea why anyone would do that, does POST urls can leak as intents on mobile. And s3 presigned urls have no other security too